tests/unittests/test_lookback_window.py

```diff
@@ -15,12 +15,12 @@ def to_dict_recursive(cls):
 
 bookmark_time = 1645046000 # epoch bookmark time
 
-@mock.patch("tap_stripe.reduce_foreign_keys", return_value = {"date": 16452804585})
-@mock.patch("tap_stripe.convert_dict_to_stripe_object", return_value = {"date": "2022-02-17T00:00:00"})
+@mock.patch("tap_stripe.reduce_foreign_keys", return_value = {"created": 16452804585})
+@mock.patch("tap_stripe.convert_dict_to_stripe_object", return_value = {"created": "2022-02-17T00:00:00"})
```

Comment:
Why we need to change this?

Reply:
Because the replication key for balance_transactions stream is `created` and we used `date`